src/main/scala/org/opennetworkinsight/netflow/FlowPreLDA.scala

```diff
@@ -23,186 +21,126 @@ object FlowPreLDA {
-      //   for example:   2016-04-21 03:58:13
-      val hourMinSecond: Array[String] = tstart.split(' ')(1).split(':') // todo: error handling if the line is malformed
-      val hour = hourMinSecond(0)
-      val min = hourMinSecond(1)
-      val sec = hourMinSecond(2)
-
-
-      val time_flow_index = 0
-      val year_flow_index = 1
-      val month_flow_index = 2
-      val day_flow_index = 3
-      val hour_flow_index = 4
-      val minute_flow_index = 5
-      val second_flow_index = 6
-      val tdur_flow_index = 7
-      val sip_flow_index = 8
-      val dip_flow_index = 9
-      val sport_flow_index = 10
-      val dport_flow_index = 11
-      val proto_flow_index = 12
-      val flag_flow_index = 13
-      val fwd_flow_index = 14
-      val stos_flow_index = 15
-      val ipkt_flow_index = 16
-      val ibyt_flow_index = 17
-      val opkt_flow_index = 18
-      val obyt_flow_index = 19
-      val input_flow_index = 20
-      val output_flow_index = 21
-      val sas_flow_index = 22
-      val das_flow_index = 23
-      val dtos_flow_index = 24
-      val dir_flow_index = 25
-      val rip_flow_index = 26
-
-
-      val buf = new StringBuilder
-      for (i <- 0 to 26) {
-        if (i == hour_flow_index) {
-          buf ++= hour
-        } else if (i == minute_flow_index) {
-          buf ++= min
-        } else if (i == second_flow_index) {
-          buf ++= sec
-        } else if (i == ipkt_flow_index) {
-          buf ++= ipkts
-        } else if (i == ibyt_flow_index) {
-          buf ++= ibyts
-        } else if (i == sport_flow_index) {
-          buf ++= sport
-        } else if (i == dport_flow_index) {
-          buf ++= dport
-        } else if (i == sip_flow_index) {
-          buf ++= srcIP
-        } else if (i == dip_flow_index) {
-          buf ++= dstIP
-        } else {
-          buf ++= "##"
-        }
-        if (i < 26) {
-          buf + ','
-        }
-      }
-      buf.toString()
-    }
+    val scoredFileExists = if (scoredFile != "") new java.io.File(scoredFile).exists else false
 
-    logger.info("Trying to read file:  " + inputPath)
-    val rawdata: RDD[String] = {
-      val flowDataFrame = sqlContext.parquetFile(inputPath)
-        .filter("trhour BETWEEN 0 AND 23 AND  " +
-          "trminute BETWEEN 0 AND 59 AND  " +
-          "trsec BETWEEN 0 AND 59")
-        .select("treceived",
-          "tryear",
-          "trmonth",
-          "trday",
-          "trhour",
-          "trminute",
-          "trsec",
-          "tdur",
-          "sip",
-          "dip",
-          "sport",
-          "dport",
-          "proto",
-          "flag",
-          "fwd",
-          "stos",
-          "ipkt",
-          "ibyt",
-          "opkt",
-          "obyt",
-          "input",
-          "output",
-          "sas",
-          "das",
-          "dtos",
-          "dir",
-          "rip")
-      flowDataFrame.map(_.mkString(","))
-    }
+    val falsePositives: DataFrame = if (scoredFileExists) {
```

Comment:
It's worth doing that change, it's something I was tempting to do too but because of the other priorities I didn't. 

Will work on it next sprint for DNS and Flow.